packages/cms/pages/resources/resource/resource.tsx

```diff
@@ -291,6 +297,20 @@ export const Table: React.FunctionComponent<TableProps> = ({ search }) => {
       {
         name: 'Actions',
         actions: [
+          {
+            name: 'Edit',
+            description: 'Edit this item',
+            icon: 'pencil',
+            type: 'icon',
+            // color: 'danger',
+            onClick: item => {
```

Comment:
Let's check if this could be a link. If it can, then let's make it one. If not, this is fine.